### PR TITLE
Update Lerna icon

### PR DESCRIPTION
**Pull request by LitoMore**
### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Update the icon according to https://github.com/lerna/lerna/issues/4088. The color was picked from the their primary color on the project homepage.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8827,8 +8827,8 @@
         },
         {
             "title": "Lerna",
-            "hex": "334155",
-            "source": "https://github.com/lerna/website/blob/dcef9c248b2b67bdd2214058beca437732690226/images/lerna-logo-dark.svg"
+            "hex": "9333EA",
+            "source": "https://user-images.githubusercontent.com/900523/173044458-fd0b57f6-6374-4265-98b5-eb8f55fe1fb3.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:8831]: I don't think we should use user uploaded images as sources like this. Where is this taken from?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json:8831]: From their GitHub readme, uploaded by their owner. There are no more links for it. It's the most accurate one.

GitHub asset links are always permanent. So it's OK to use the `user-images.githubusercontent` link.

**Comment by LitoMore** [comment]
@uncenter Can I get an approval? Thanks~

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8827,8 +8827,8 @@
         },
         {
             "title": "Lerna",
-            "hex": "334155",
-            "source": "https://github.com/lerna/website/blob/dcef9c248b2b67bdd2214058beca437732690226/images/lerna-logo-dark.svg"
+            "hex": "9333EA",
```
Comment (uncenter) [inline _data/simple-icons.json:8830]:
```suggestion
            "hex": "C084FC",
```

The color of the favicon is a darker purple which I'm guessing is what you used here, but on the website itself the primary purple used is `#c084fc`. Shall we use that instead?

File: icons/lerna.svg
Comment (uncenter) [inline icons/lerna.svg:1]: Looks good!

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (LitoMore) [reply _data/simple-icons.json:8830]: Sounds good. Let's use the more colorful one.

**Review by uncenter — APPROVED** [review]

**Pull request merged**